openjdk/playlist.xml

```diff
@@ -1475,6 +1475,10 @@
 	<test>
 		<testCaseName>jdk_jfr</testCaseName>
 		<disables>
+			<disable>
+				<comment>https://github.com/adoptium/aqa-tests/issues/3115#issuecomment-2256561374</comment>
+				<vendor>eclipse</vendor>
```

Comment:
needs a `<platform>` element to specify just disabling for a particular platform